make/autoconf/jdk-version.m4

```diff
@@ -109,6 +109,33 @@ AC_DEFUN_ONCE([JDKVER_SETUP_JDK_VERSION_NUMBERS],
   fi
   AC_SUBST(MACOSX_BUNDLE_ID_BASE)
 
+  # The launcher name, if any
+  AC_ARG_WITH(launcher-name, [AS_HELP_STRING([--with-launcher-name],
+      [Set launcher name. Part of the -version and -fullversion output. @<:@not specified@:>@])])
+  if test "x$with_launcher_name" = xyes; then
+    AC_MSG_ERROR([--with-launcher-name must have a value])
+  elif [ ! [[ $with_launcher_name =~ ^[[:print:]]*$ ]] ]; then
+    AC_MSG_ERROR([--with-launcher-name contains non-printing characters: $with_launcher_name])
+  elif test "x$with_launcher_name" != x; then
+    # Only set LAUNCHER_NAME if '--with-launcher-name' was used and is not empty.
+    # Otherwise we will use the value from "version-numbers" included above.
+    LAUNCHER_NAME="$with_launcher_name"
+  fi
+
+  # The product name, if any
+  AC_ARG_WITH(product-name, [AS_HELP_STRING([--with-product-name],
+      [Set product name. Among others, used to set the 'java.runtime.name'
+       system property. @<:@not specified@:>@])])
```

Comment:
The help test should not have internal line breaks; suggest:
```
      [Set product name. Among other uses, defines the 'java.runtime.name' system property. @<:@not specified@:>@])])
```

Reply:
The line break was removed, but the wording was not updated.

Reply:
Updated now.